src/test/java/edu/ucsb/courses/controllers/ScheduleItemControllerTests.java

```diff
@@ -0,0 +1,487 @@
+package edu.ucsb.courses.controllers;
+
+import com.auth0.jwt.JWT;
+import com.auth0.jwt.interfaces.DecodedJWT;
+import edu.ucsb.courses.config.SecurityConfig;
+import edu.ucsb.courses.documents.Course;
+import edu.ucsb.courses.documents.Section;
+import edu.ucsb.courses.documents.TimeLocation;
+import edu.ucsb.courses.entities.Schedule;
+import edu.ucsb.courses.entities.ScheduleItem;
+import edu.ucsb.courses.repositories.ScheduleItemRepository;
+import edu.ucsb.courses.repositories.ScheduleRepository;
+import edu.ucsb.courses.repositories.ArchivedCourseRepository;
+import org.apache.commons.lang3.arch.Processor;
+import org.codehaus.jackson.map.ObjectMapper;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.context.annotation.Import;
+import org.springframework.http.HttpHeaders;
+import org.springframework.security.test.context.support.WithMockUser;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.MvcResult;
+
+import javax.swing.text.html.Option;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.*;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+import static org.springframework.security.test.web.servlet.request.SecurityMockMvcRequestPostProcessors.csrf;
+
+
+// @Import(SecurityConfig.class) applies the security rules 
+// so that /api/public/** endpoints don't require authentication.
+// Otherwise you may get authorization errors when running the test
+
+@WebMvcTest(value = ScheduleItemController.class)
+@WithMockUser
+//@Import(SecurityConfig.class)
```

Comment:
If `@Import(SecurityConfig.class)` isn't necessary, can we remove these comments?

Reply:
I think that after they move the /api from /api/public to /api/member where it *should* be, they are going to need those lines of code.  So I would suggest that they NOT remove the comments and that they restore line 47.